src/basic/terminal-util.h

```diff
@@ -111,6 +111,10 @@ typedef enum ColorMode {
         /* Only 256 colors. */
         COLOR_256 = 256,
 
+        /* COLORTERM default value truecolor for terminator, 16,777,216 colors. 24-bit color Value. */
+        /* Other Possible value gnome-terminal */
+        COLOR_TERM = 24,
```

Comment:
How about to rename `COLOR_TERM` -> `COLOR_16777216` or `COLOR_24BIT`?
And, the assigned value should be 16777216, instead of 24.

Reply:
Sure